### PR TITLE
Fix consider-using-dict-items warnings

### PR DESCRIPTION
**Pull request by brianphaley**
Fix consider-using-dict-items warnings and start enforcing in pylintrc.

### CODE REVIEW

File: hotsos/core/plugins/storage/ceph.py [hunk 1]
```diff
@@ -206,9 +206,9 @@ def crushmap_mixed_buckets(self):
 
         bad_buckets = []
         buckets = self._build_buckets_from_crushdump(self.osd_crush_dump)
-        # check all bucket
-        for bid in buckets:
-            items = buckets[bid]["items"]
+        # check all buckets
+        for _, bdict in buckets.items():
```
Comment (dosaboy) [inline hotsos/core/plugins/storage/ceph.py:210]: this should be:

```
for bdict in buckets.values():
```

File: hotsos/core/plugins/system/system.py [hunk 2]
```diff
@@ -202,21 +202,21 @@ def ubuntu_pro_status(self):
             result["status"] = "attached"
             result["services"] = {}
 
-            for sid in ssects:
+            for _, values in ssects.items():
```
Comment (dosaboy) [inline hotsos/core/plugins/system/system.py:205]: this should be:
```
for values in ssects.values():
```

File: hotsos/core/plugins/system/system.py [hunk 3]
```diff
@@ -202,21 +202,21 @@ def ubuntu_pro_status(self):
...
                         "entitled": v.get(2),
                         "status": v.get(3)
                     }
-                    for v in ssects[sid]
+                    for v in values
                     if v.tag == service_status_seqdef.body_tag
                 }}
 
-            for sid in asects:
+            for _, values in asects.items():
```
Comment (dosaboy) [inline hotsos/core/plugins/system/system.py:215]: this should be:

```
for values in asects.values():
```

File: hotsos/core/plugins/rabbitmq/report.py [hunk 4]
```diff
@@ -73,11 +73,10 @@ def skewed_nodes(self):
             # Report the node with the greatest skew of queues/vhost
             if skewed_queue_nodes:
                 max_node = None
-                for node_name in skewed_queue_nodes:
+                for node_name, count in skewed_queue_nodes.items():
```
Comment (pponnuvel) [inline hotsos/core/plugins/rabbitmq/report.py:76]: Not related to the change but could simplify this.

This block:
```
                max_node = None
                for node_name, count in skewed_queue_nodes.items():
                    if max_node is None:
                        max_node = node_name
                    elif count >= skewed_queue_nodes[max_node]:
                        max_node = node_name
```

can be replaced with:
```
    max_node = max(skewed_queue_nodes, key=skewed_queue_nodes.get)
```

Comment (pponnuvel) [reply hotsos/core/plugins/rabbitmq/report.py:76]: Here you go: https://github.com/canonical/hotsos/pull/824

Apologies in advance, @brianphaley, for causing a conflict ;-)

**Review by dosaboy — APPROVED** [review]
lgtm, thanks!

**Pull request merged**